### PR TITLE
feat: add export transaction and error feat, fix format username bug

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

**Comment by kleyow** [comment]
Not sure why the CI checks don't pass.

File: ui-tests/tests/src/tests/TransferDashboardPage.test.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      await t.click(LoginPage.loginButton)
+    }
+  });
+
+test.meta({
+  ID: '',
+  STORY: '',
+  description: '',
+})('Can download excel spreadsheet of transfers in basic search', async (t) => {
+  const expectedFilePath = `${process.env.HOME}` + '/Downloads/' + `Payment_Manager_Transfers_${new Date().toDateString()}.xlsx`;
```
Comment (sri-miriyala) [inline ui-tests/tests/src/tests/TransferDashboardPage.test.ts]: can /Downloads/ be part of `` instead of as a separate '' @kleyow

Comment (kleyow) [reply ui-tests/tests/src/tests/TransferDashboardPage.test.ts:24]: done.

Comment (kleyow) [reply ui-tests/tests/src/tests/TransferDashboardPage.test.ts:24]: done.

**Review by sri-miriyala — APPROVED** [review]
Looks good

**Pull request merged**